docs/samples/python/sample_data/ground_truth.json

```diff
@@ -0,0 +1,143 @@
+{
+    "sample_data/0_ORIGINAL.dcm": {
+        "0": {
```

Comment:
Good point. I honestly just forgot that using a list of objects was an option.

I'll see if I can update the code to use lists of objects instead of 100% dictionaries. This may result in some changes in the `DicomImageRedactorEngine` files as well.

Reply:
Everything updated! Now using lists of dictionaries instead of keeping manual string indexes in a larger dictionary.